components/inflector.rst

```diff
@@ -0,0 +1,53 @@
+.. index::
+   single: Inflector
+   single: Components; Inflector
+
+The Inflector Component
+=======================
+
+    The Inflector component converts English words between singular and plural forms.
```

Comment:
Afaik it's always capitalised in the English language: http://www.oxfordlearnersdictionaries.com/definition/english/english_2